python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/test_ecs_client.py

```diff
@@ -4,107 +4,62 @@
 import pytest
 from dagster.core.test_utils import instance_for_test
 from dagster_aws.ecs import client
-from moto import mock_ecs, mock_logs
+from moto import mock_sts
 
 
-@mock_ecs
-@mock_logs
-def test_set_cluster():
-    log_client = boto3.client("logs", region_name="us-east-2")
-    ecs_client = boto3.client("ecs", region_name="us-east-2")
-    ecs_client.create_cluster(clusterName="test_clust")
-    testclient = client.ECSClient(
+@pytest.fixture
+def test_client(mock_ecs_client, mock_ecs_cluster):  # pylint: disable=unused-argument
```

Comment:
move this up to ecs_tests/conftest.py? is `mock_ecs_cluster` here for the side effect of creating the cluster?

Reply:
Now that you mention it, I might collapse conftest back into the test file. The ECS resources are pretty under-tested so originally I was planning to add some additional test coverage. But if that happens, it'll be in a separate PR anyway.

And yup, `mock_ecs_cluster` is included for the side effect of setup/teardown of the cluster for any test that uses the `test_client`.